inlong-manager/manager-schedule/src/test/java/org/apache/inlong/manager/schedule/airflow/AirflowContainerEnv.java

```diff
@@ -0,0 +1,124 @@
+                        "airflow.providers.fab.auth_manager.api.auth.backend.basic_auth");
+        environment.start();
+        log.info("All services are up and running.");
+        loadTestDAG();
+        waitForDAGLoad("dag_cleaner");
+    }
+
+    private static void loadTestDAG() {
+        // After the DAG file is created, the scheduler will regularly scan the DAG file directory and
+        // then load it into memory for scheduling. In order to quickly test the update and unregister, two
+        // test DAGs need to be loaded at the beginning.
+        String airflowSchedulerName = "airflow-scheduler_1";
+        log.info("airflow-scheduler name : " + airflowSchedulerName);
+        Optional<ContainerState> container = environment.getContainerByServiceName(airflowSchedulerName);
+        log.info("container : " + container.isPresent());
+        if (container.isPresent()) {
+            ContainerState airflowScheduler = container.get();
+            Path dagPath1 = Paths.get("src/test/resources/airflow/dag_cleaner.py").toAbsolutePath();
+            Path dagPath2 = Paths.get("src/test/resources/airflow/dag_creator.py").toAbsolutePath();
+            Path dagPath3 = Paths.get("src/test/resources/airflow/testGroup_cron.py").toAbsolutePath();
+            Path dagPath4 = Paths.get("src/test/resources/airflow/testGroup_normal.py").toAbsolutePath();
+            airflowScheduler.copyFileToContainer(MountableFile.forHostPath(dagPath1),
+                    "/opt/airflow/dags/dag_cleaner.py");
+            airflowScheduler.copyFileToContainer(MountableFile.forHostPath(dagPath2),
+                    "/opt/airflow/dags/dag_creator.py");
+            airflowScheduler.copyFileToContainer(MountableFile.forHostPath(dagPath3),
+                    "/opt/airflow/dags/testGroup_cron.py");
+            airflowScheduler.copyFileToContainer(MountableFile.forHostPath(dagPath4),
+                    "/opt/airflow/dags/testGroup_normal.py");
+            try {
+                log.info("has file: "
+                        + airflowScheduler.execInContainer("bash", "-c", "ls /opt/airflow/dags/").getStdout());
+            } catch (Exception ignored) {
+
+            }
+        }
+    }
+
+    public static void waitForDAGLoad(String dagId) {
+        int total = 10;
+        // Waiting for Airflow to load the initial DAG
+        while (total > 0) {
+            String credential = okhttp3.Credentials.basic(AIRFLOW_USERNAME, AIRFLOW_PASSWORD);
+            Request request = new Request.Builder()
+                    .url(BASE_URL + "/api/v1/dags/" + dagId + "/details")
+                    .header("Authorization", credential)
+                    .build();
+            try (Response response = httpClient.newCall(request).execute()) {
+                if (response.code() == 200) {
+                    break;
+                }
+
+            } catch (Exception e) {
+                log.error(e.getMessage());
```

Comment:
Don't just print `errMessage`, print out the complete exception stack.

Reply:
done.